docs/content/user-guide/api-key-management.md

```
@@ -157,9 +157,69 @@ curl -sS "${MAAS_API_URL}/maas-api/v1/api-keys/${KEY_ID}" \
+
+| Unit | Examples | Description |
+|------|----------|-------------|
+| `d` | `"1d"`, `"30d"`, `"90d"`, `"1.5d"` | Days (24 hours each) |
+| `h` | `"1h"`, `"6h"`, `"24h"`, `"2.5h"` | Hours |
+| `m` | `"30m"`, `"90m"`, `"120m"` | Minutes |
+| `s` | `"60s"`, `"3600s"` | Seconds |
+| Combined | `"2h30m"`, `"1h45m30s"` | Multiple units (no spaces) |
+
+**Numeric format (seconds):**
+
+When passed as a JSON number (not a string), the value is interpreted as **seconds**. E.g. the following `expiresIn: 86400` is one day (24 * 60 * 60):
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)